pyproject.toml

```diff
@@ -37,7 +37,12 @@ groq = ["groq"]
 cohere = ["cohere"]
 google-generativeai = ["google-generativeai"]
 examples = ["transformers", "datasets", "scipy", "torch", "pandas"]
-chat = ["fastapi"]
+chat = [
+    "fastapi",
+    "websockets>=13.1",
+    "rel",
```

Comment:
FastAPI and uvicorn make sense.

Can we use `aiohttp` instead of `websockets`? `rel` seems to be not used anywhere